automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala

```diff
@@ -41,8 +44,18 @@ trait UserFixtures extends CleanUp { self: WebBrowserSpec with TestSuite =>
                                                 (testCode: (T) => Any)
                                                 (implicit webDriver: WebDriver): Unit = {
     withSignIn(user, {
-      new SignInPage(Config.FireCloud.baseUrl).open
-      executeScript(s"window.forceSignedIn('${user.makeAuthToken().value}')")
+      val openedPage: SignInPage = new SignInPage(Config.FireCloud.baseUrl).open
+      retry[T](1.seconds, 60.seconds) ({
+          openedPage.signInButton.awaitVisible()
+          executeScript(s"window.forceSignedIn('${user.makeAuthToken().value}')")
+          logger.info("Executed JS forceSignedIn")
+          try {
+            await.ready(page, 30)
```

Comment:
I'd rather not have to wait 30 seconds if nothing is happening. However, we don't want to cut the wait too short because the workspace list page _could_ be loading.

Maybe instead we can check for the absence of the sign-in button. Something like:

```
await condition (!openedPage.isActive, 5)
```

with this in `SignInPage`:

```
  def isActive: Boolean = signInButton.isVisible
```

That way we retry if the sign-in button doesn't go away within 5 seconds. I haven't tested this so I'm not 100% sure it works.